src/test/suite/fixtures/recorded/languages/scala/chuckTypeRed.yml

```diff
@@ -0,0 +1,33 @@
+languageId: scala
```

Comment:
We usually define type to expand to the entire type expression, rather then getting trapped in a subcomponent of the type, so I'd be inclined to change the matcher